firewood/src/shale/mod.rs

```diff
@@ -171,12 +184,15 @@ impl<T: Storable> Deref for Obj<T> {
 /// User handle that offers read & write access to the stored [ShaleStore] item.
 #[derive(Debug)]
 pub struct ObjRef<'a, T: Storable> {
+    /// WARNING:
+    /// [`Self::inner`] should only be `None` when calling `into_inner` and `drop`.
```

Comment:
Is this true? Don't we panic if we call `into_inner` and `inner` is `None`?

Reply:
🤔, I think I need to more specific. Should only be set to `None` inside `drop` or _after_ calling `into_inner`. Will update.

Reply:
Updated